packages/gatsby-theme-docs/src/components/b2b-tag.js

```diff
@@ -0,0 +1,27 @@
+import PropTypes from 'prop-types';
+import { PlanTag } from './plan-tag';
+import { Icons } from '@commercetools-docs/ui-kit';
+
+/**
+ * This component stays as shorthand of the <PlanTag /> component. The configuration
+ * is hardcoded to display B2B tags.
+ */
+export const B2B_HOVER_TEXT = 'TBD';
```

Comment:
the B2B hover text needs to be provided. @melindafekete do you have some content to add here?
We're talking about the text that the user will see when hovering over a B2B tag.
For beta tag the text is: _This feature is marked as beta and is subject to change. Use with caution_.

Reply:
@nirusuu do you have any suggestion for what we should display for the B2B tag hover text? thanks!

Reply:
I suggest we reuse the same copypasta we have used before in the docs: 
> This feature is only available as part of Composable Commerce for B2B and will be subject to additional terms and pricing.

Reply:
@gabriele-ct  I'm a bit confused.  We chose to implement a generic "plan tag" that can then be configured in the content repo with what plans with what descriptions are available. This should not happen here because this is open source, we would not be able to launch any business model changes without silently going public here.

Reply:
> I'm a bit confused. We chose to implement a generic "plan tag" that can then be configured in the content repo with what plans with what descriptions are available. This should not happen here because this is open source, we would not be able to launch any business model changes without silently going public here.

@nkuehn 
`<Beta>` and `<B2B>` components are shorthands wrapping the PlanTag component. We don't want the editors to configure the same Beta or B2B tag over an over again. If hover text will be the same, as it is for Beta, I think this is the pattern we should follow

Reply:
"over and over again" is putting the configuration once into the base theme in the docs repo.  One place to put it.

Reply:
thank you @nkuehn for clarifying this additional requirement. Do we want this pattern to be implemented for Beta tag too?
Meaning that the siteMetadata will look like this:
```
siteMetadata: {
    betaHoverText: 'This feature is marked as beta and is subject to change. Use with caution.'
    betaLink: '/../offering/api-contract#public-beta',
    b2bHoverText: 'This feature is only available as part of Composable Commerce for B2B and will be subject to additional terms and pricing.'
    b2bLink: 'some link to the beta page'
  },
```